scripts/simplify_network.py

```diff
@@ -620,6 +765,20 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    p_threshold_drop_isolated = max(
+        0.0, cluster_config.get("p_threshold_drop_isolated", False)
```

Comment:
The default should be again 0.0 instead of False.
Probably it has not practical difference but it may be better

Reply:
Agree, done